charging_current_range_test.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+import charging_current_range
+class RangeBucketTest(unittest.TestCase):
+    def test_range_details(self):
+        # self.assertTrue(getRangeDetails([4, 5, 1, 2, 2, 1, 1,2])==8)
+        self.assertTrue(charging_current_range.getRangeDetails([4, 5]) == 2)
```

Comment:
there is a different reading (number of elements per range) for each range. you have test cases for different ranges, but output doesn't seem to be as expected.
for ex, shouldn't the output for (2, 3, 4, 6, 7]) be  2-4, 3 ; 6-7,2 ??